src/Microsoft.IdentityModel.Logging/LogHelper.cs

```diff
@@ -333,22 +333,22 @@ public static void LogWarning(string message, params object[] args)
                 message = format;
 
             if (IdentityModelEventSource.Logger.IsEnabled(eventLevel, EventKeywords.All))
-                IdentityModelEventSource.Logger.Write(eventLevel, innerException, message);
+                IdentityModelEventSource.Logger.Write(eventLevel, innerException!, message);
```

Comment:
same question, how do we know innerException is not null?

Reply:
the change is rather in Logger.Write to accept null

Reply:
I don't think so. The args can be null, but I don't think we'll get any nullable warnings with the code as-is.

Reply:
@jmprieur  Is correct, I took off the warning during development to deal with another error and forgot to take it off to add the nullable arg to the logger.Write

Reply:
Made arg nullable